types/database/schema/PostNotificationChannelList.types.ts

```diff
@@ -61,6 +61,8 @@ export interface PostNotificationChannelListData {
   isSeen: boolean;
   totalComment: number;
   isOwnPost: boolean;
+  isOwnAnonymousPost: boolean;
+  isOwnSignedPost: boolean;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `isOwnAnonymousPost` and `isOwnSignedPost` fields seems to be a good way to differentiate between different types of posts. However, it's important to ensure that these new fields are properly initialized and updated wherever this interface is used. Also, consider the case where both `isOwnAnonymousPost` and `isOwnSignedPost` could be true at the same time. If such a scenario is not valid in your application logic, you might want to add checks to prevent this from happening.


<!-- This is an auto-generated comment by OSS CodeRabbit -->